frontend/src/components/forms/CreateLog.tsx

```diff
@@ -331,14 +322,17 @@ const CreateLog = () => {
                       placeholder="Building No."
                       onChange={handleBuildingChange}
                       styles={{
```

Comment:
The changes in this file are a combination of changes, so I'll add the explanation for the main changes in the `selectStyle.tsx` file. Update the styles to only use selectStyle
```suggestion
                      styles={selectStyle}
```

Reply:
oops I forgot to add the explanation but basically, we should keep all the styling in `selectStyle`. The error border rendering was removed in my suggestion because it was something I added in but it wasn't based off of any designs.